synthesizer/src/block/transactions/mod.rs

```diff
@@ -64,87 +66,63 @@ impl<'a, N: Network> FromIterator<&'a Transaction<N>> for Transactions<N> {
 impl<N: Network> Transactions<N> {
     /// Returns `true` if the transactions contains the given transition ID.
     pub fn contains_transition(&self, transition_id: &N::TransitionID) -> bool {
-        match cfg!(feature = "parallel") {
-            true => self.par_values().any(|tx| tx.contains_transition(transition_id)),
-            false => self.values().any(|tx| tx.contains_transition(transition_id)),
+        match cfg!(feature = "serial") {
+            false => self.par_values().any(|tx| tx.contains_transition(transition_id)),
+            true => self.values().any(|tx| tx.contains_transition(transition_id)),
         }
     }
 
     /// Returns `true` if the transactions contains the given serial number.
     pub fn contains_serial_number(&self, serial_number: &Field<N>) -> bool {
-        match cfg!(feature = "parallel") {
-            true => self.par_values().any(|tx| tx.contains_serial_number(serial_number)),
-            false => self.values().any(|tx| tx.contains_serial_number(serial_number)),
+        match cfg!(feature = "serial") {
+            false => self.par_values().any(|tx| tx.contains_serial_number(serial_number)),
+            true => self.values().any(|tx| tx.contains_serial_number(serial_number)),
         }
     }
 
     /// Returns `true` if the transactions contains the given commitment.
     pub fn contains_commitment(&self, commitment: &Field<N>) -> bool {
-        match cfg!(feature = "parallel") {
-            true => self.par_values().any(|tx| tx.contains_commitment(commitment)),
-            false => self.values().any(|tx| tx.contains_commitment(commitment)),
+        match cfg!(feature = "serial") {
+            false => self.par_values().any(|tx| tx.contains_commitment(commitment)),
+            true => self.values().any(|tx| tx.contains_commitment(commitment)),
```

Comment:
Why not use the `cfg_values` macro for these instances?

Reply:
Yeah this tracks